chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,6 +150,12 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
+            # we want disks that are multipath but not if they are holders
+            # of a multipath
+            if d.multipath?
```

Comment:
Calling them "mutually exclusive" was wrong. It may happend that `d.multipath?` and `d.held_by_multipath?` are false.